documentation/milestones/individual_milestones/search_mvp.md

```diff
@@ -0,0 +1,218 @@
+  - To receive an incentive if I participate in user research for simpler.grants.gov, so that I am fairly compensated for the time that I invest in testing out new features and answering questions
+  - Filters and search terms to be added to the url, so that I can save or share the link to a particular set of search results
+- As a **project maintainer**, I want:
+  - To be notified when simpler.grants.gov experiences downtime, so that I can quickly troubleshoot the issue with minimal impact on end users
+  - The system architecture we create for this deliverable to support future search features as well, so that we don't need to discard and re-build critical infrastructure
+
+## Technical description
+
+### Search API
+<!-- Optional -->
+
+The ability to search and filter opportunities should be provided through an API endpoint, which allows the user to:
+
+- Search the list of opportunities with open ended key words
+- Filter the search results by *at least* one structured field (e.g. award date, opportunity type, etc.)
+- Sort the search results based on the value of *at least*
+
+Through the design and development of this API endpoint, the team should answer the following questions and record the decisions in one or more ADRs:
+
+- How should search and filter criteria be passed to the API endpoint used for search?
+- How should we support basic logic such as greater than, less than, as well as and/or/not conditions?
+- Which fields should support filtering and sorting?
+- Do we use an existing search service (e.g. Elasticsearch) or build our own?
+
+### Search Page
+
+The primary page we'll need to deliver for the user interface (UI) is a search page, which should allow users to:
+
+- Enter key words to search the list of funding opportunities
+- Filter search results by *at least* one structured field (e.g. award date, opportunity type, etc.)
+- View a common set of fields for each opportunity returned by the search (e.g. title, award date, summary, etc.)
+- Sort the set of results returned by *at least* one structured field (e.g. award date, title, etc.)
+
+### Opportunity Listing Page
+
+The secondary page we want to deliver within the user interface (UI) is an opportunity listing page, which should:
+
+- Display a consistent list of metadata about each opportunity
+- Use only fields that are available in the existing grants.gov cloud database
+- Link to the corresponding opportunity in the existing grant.gov interface
+
+**Note:** This is a stretch goal for this deliverable and may be substituted with a direct link to the grants.gov page for a given opportunity from the search results.
+
+### Definition of done
+<!-- Required -->
+
+- [ ] Basic requirements:
+  - [ ] Code is deployed to `main` & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] Translations are live in PROD (if necessary)
```

Comment:
Aaron will be reviewing Monday but not sure about how translations may work cc @acouch